<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMailboxForm</class>
 <widget class="QWidget" name="AddMailboxForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>245</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>462</width>
    <height>245</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New mailbox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="email_layout">
       <item>
        <widget class="QLabel" name="email">
         <property name="text">
          <string>                             email:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="email_edit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="password_layout">
       <item>
        <widget class="QLabel" name="password">
         <property name="text">
          <string>                     Password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="password_edit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="incoming_server_type_layout">
       <item>
        <widget class="QLabel" name="inc_server_type_label">
         <property name="text">
          <string>  Incoming server type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="incoming_server_type">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>IMAP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>POP3</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="incoming_server_layout">
       <item>
        <widget class="QLabel" name="incoming_server">
         <property name="text">
          <string>   Incoming mail server:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="incoming_server_edit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="outgoing_server_layout">
       <item>
        <widget class="QLabel" name="outgoing_server">
         <property name="text">
          <string>   Outgoing mail server:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="outgoing_server_edit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
